dolphin/drivers/manager.py

```diff
@@ -19,6 +19,7 @@
 
 DRIVER_MAPPING = {
     "fake_storage": "dolphin.drivers.fake_storage.FakeStorageDriver"
+    "vmax_storage": "dolphin.drivers.fake_storage.VMAXStorageDriver"
```

Comment:
lets make it dell_emc - > vmax_storage": "dolphin.drivers.dell_emc.VMAXStorageDrive

Reply:
Done